browser-extension/common/src/popup/components/Popup/PopupHeader.tsx

```diff
@@ -1,34 +1,25 @@
 import React, { useCallback } from "react";
-import { Button, Dropdown, Menu, Row, Typography } from "antd";
+import { Button, Col, Row, Switch, Typography, Tooltip } from "antd";
 import config from "../../../config";
-import ArrowDown from "../../../../resources/icons/arrowDown.svg";
 import { EVENT, sendEvent } from "../../events";
 
 interface PopupHeaderProps {
   isExtensionEnabled: boolean;
   handleToggleExtensionStatus: () => void;
+  initPauseToolip: boolean;
```

Comment:
Instead of adding a new state variable for tooltip, we can default keep the `isExtensionEnabled` as true

```diff
- const [isExtensionEnabled, setIsExtensionEnabled] = useState<boolean>(false);
+ const [isExtensionEnabled, setIsExtensionEnabled] = useState<boolean>(true);
```